here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -8,6 +8,8 @@ servers:
   #- url: "https://naksha-e2e.ext.mapcreator.here.com/"
   #  description: "E2E"
   - url: "https://naksha-dev.ext.mapcreator.here.com/"
```

Comment:
Knowing our consumer (UI) has now moved to v2, we can now remove "DEV-Old" entry completely from Swagger.

Reply:
fixed